nifty-client/pom.xml

```diff
@@ -72,6 +72,13 @@
         </dependency>
 
         <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative-boringssl-static</artifactId>
+            <version>1.1.33.Fork17</version>
+            <classifier>${os.detected.classifier}</classifier>
```

Comment:
This is the wrong approach -- it is only including the dependency for the build platform. For example, if you build this on Mac, someone running on Linux will have the wrong version. The proper way to do JNI is to include the libraries for all supported platforms and then unpack and load the correct version at runtime.